exercises/java-exceptions/src/main/java/com/cbfacademy/FileExtension.java

```diff
@@ -0,0 +1,15 @@
+package com.cbfacademy;
+
+public class FileExtension {
+    public static int check(String filename)throws NullorEmptyException {
+        if(filename == null || filename ==""){
+            throw new NullorEmptyException("file name cannot be null or empty");
+        }
+        else if(filename.endsWith(".java")){
```

Comment:
This can be on the same line as the curly brace on line 7. You don't need a new line before the `else if`

Reply:
Easiest way to deal with most of these formatting issues is to simply let your editor take care of it. You can quickly clean up a file by selecting View > Command Palette > Format Document (or using the equivalent keyboard shortcut)